platform/ui/src/views/StudyList/components/StudyListTable.js

```diff
@@ -9,7 +11,29 @@ import InstancesActive from '../../../assets/icons/instances-active.svg';
 import InstancesInactive from '../../../assets/icons/instances-inactive.svg';
 import LaunchInfo from '../../../assets/icons/launch-info.svg';
 
-const TableRow = () => {
+const getStudyModalities = series => {
```

Comment:
Assume you are getting the data in the correct format. No need to compute it here.

Reply:
I have changed the JSON mock file to accomplish the correct data. Now the component is only receiving and displaying the data as it is.